Document usage of suspense option of next/dynamic (#28210)
vercel/next.js: docs/advanced-features/dynamic-import.md

export default Home
```

## With suspense

> Option `suspense` let you to load component lazily as `React.lazy` and `<Suspense>` with React 18. Note that it only works on client side and server side will render `fallback` component yet. Fully SSR support in concurrent mode is working in progress.

Reviewer: ```suggestion
Option `suspense` allows you to lazy-load a component, similar to `React.lazy` and `<Suspense>` with React 18. Note that it only works on client-side or server-side with `fallback`. Full SSR support in concurrent mode is still a work-in-progress.
```

I'm not sure if I captured the true meaning here. Does it work with `fallback`?
